src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const mediumDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return days * price;
+  }
+
+  if (days >= 3 && days < 7) {
+    return (days * price) - mediumDiscount;
+  }
+
+  if (days >= 7) {
+    return (days * price) - bigDiscount;
+  }
```

Comment:
you can move check with days >= 3 under days >= 7 and then you can remove days < 7

and you don't need `days < 3`, you can return `days * price` if all previous checks were false